stock_inventory_revaluation/models/stock_inventory_revaluation.py

```diff
@@ -0,0 +1,407 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models, _
+import openerp.addons.decimal_precision as dp
+import time
+from openerp.exceptions import Warning as UserError
+
+_STATES = [
```

Comment:
There's no need to extract this in v8, the API provides means to add new states. Or would you like to make it a function others can redefine?